packages/cyberstorm/src/components/PackageCard.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/PackageCard.module.css";
+import { MetaItem } from "./MetaItem";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faDownload,
+  faThumbsUp,
+  faHardDrive,
+} from "@fortawesome/free-solid-svg-icons";
+import { Tag } from "./Tag";
+
+const defaultImageSrc = "";
+
+export interface PackageCardProps {
+  packageName: string;
+  description?: string;
+  imageSrc?: string;
+  downloadCount: string;
+  likes: string;
+  size: string;
+  author?: string;
+  link: string;
+  lastUpdated?: string;
+  isPinned?: boolean;
+  isNsfw?: boolean;
+  isDeprecated?: boolean;
+  packageCardStyle: "default";
+  categories: string[];
+}
+
+/**
+ * Cyberstorm PackageCard component
+ */
+export const PackageCard: React.FC<PackageCardProps> = (props) => {
+  const {
+    packageName,
+    description,
+    imageSrc,
+    downloadCount,
+    likes,
+    size,
+    author,
+    link,
+    lastUpdated,
+    isPinned,
+    isNsfw,
+    isDeprecated,
+    packageCardStyle,
+    categories,
+  } = props;
+
+  const authorLink = ""; //TODO: author link
+
+  return (
+    <div
+      /* TS is not aware of defaultProps of function components. */
+      /* eslint-disable-next-line @typescript-eslint/no-non-null-assertion */
+      className={`${styles.root}
+      ${getStyle(packageCardStyle)}`}
+    >
+      <div className={styles.imageWrapper}>
+        <img
+          src={imageSrc ? imageSrc : defaultImageSrc}
+          className={styles.image}
+          alt={"package card of" + packageName}
+        />
+      </div>
+
+      <div className={styles.content}>
+        <a href={link} className={styles.name}>
+          {packageName}
+        </a>
+
+        <div className={styles.author}>
+          <span className={styles.author_prefix}>by </span>
+          <a className={styles.author_label} href={authorLink}>
+            {author}
+          </a>
+        </div>
+
+        <p className={styles.description}>{description}</p>
+
+        <div className={styles.categoryWrapper}>
+          {getCategoryTagList(categories)}
+        </div>
+      </div>
+
+      <div className={styles.footer}>
+        <p className={styles.lastUpdated}>{"Last updated: " + lastUpdated}</p>
```

Comment:
There's this [RelativeTime component](https://github.com/thunderstore-io/thunderstore-ui/blob/master/packages/components/src/components/RelativeTime.tsx) we might want to copy-paste from the old components to Cyberstorm too.

Reply:
Added a TODO about this, I think it's not a blocking issue.
This should be faced at the latest when hooking that time data to this component.

Reply:
Added the check